api/src/main/resources/rdr.yaml

```diff
@@ -1,694 +1,654 @@
-#This file provide the end points for interaction with RDR
-
-
-swagger: '2.0'
+# This file defines the interface we share with the RDR team for the "RDR Export" process
+# which ultimately populates the Researcher Directory.
+# See https://github.com/all-of-us/workbench/wiki/Researcher-Directory-(RDR-export) for more detail.
 
+openapi: 3.0.1
 info:
   title: RDR
   description: |
     Service for RDR Interactions.
   version: "0.1"
-#The host will get updated as per the environment it will be pmi-drc-api-test for dev and test
-host: "pmi-drc-api-test.appspot.com"
-schemes:
-  - "https"
-basePath: /
-
-securityDefinitions:
```

Comment:
See line 645 for the new version `securitySchemes` - also I recommend turning off whitespace diffs!